DESCRIPTION

```diff
@@ -1,17 +1,27 @@
 Package: CTUHelpR
-Title: What the Package Does (One Line, Title Case)
-Version: 0.0.2
+Title: Small helper functions used in the CTU Basel
+Version: 0.0.3
 Author:
-    c(person(given = "First",
-             family = "Last",
+    c(person(given = "CTUBasel",
+             role = c("cph")),
+      person(given = "Pascal",
+             family = "Benkert",
+             role = c("aut"),
+             email = "pascal.benkert@usb.ch"),
+      person(given = "Patrick R.",
+             family = "Wright",
              role = c("aut", "cre"),
-             email = "first.last@example.com"),
-      person(given = "John",
-             family = "Smith",
+             email = "patrick.wright@usb.ch"),
+      person(given = "Inessa",
+             family = "Kraft",
+             role = c("aut"),
+             email = "inessa.kraft@usb.ch"),
+      person(given = "Milica",
+             family = "Markovic",
              role = c("aut"),
-             email = "john.smith@example.com"))
```

Comment:
This is great! I have to change this part in the sum() as well. But was it planned to leave the example email addresses (Patrick, you)?